samples/react-sample-app/src/AuthProvider.js

```diff
@@ -34,6 +34,8 @@ export default C =>
                     return redirect
                         ? msalApp.acquireTokenRedirect(request)
                         : msalApp.acquireTokenPopup(request);
+                } else {
+                    console.error('Non-interactive error:', error.errorCode)
```

Comment:
Timeout errors were previously not bubbling up to the call site in the application.